common/persistence/sql/sqlplugin/postgres/plugin.go

```diff
@@ -91,3 +98,8 @@ func (d *plugin) createDBConnection(cfg *config.SQL) (*sqlx.DB, error) {
 	db.MapperFunc(strcase.ToSnake)
 	return db, nil
 }
+
+// TODO: implement postgres specific support for TLS
+func registerTLSConfig(cfg *config.SQL) error {
+	return fmt.Errorf("tls for postgres has not been implemented")
```

Comment:
I'm not too familiar with how postgres tls works so I left this unimplemented, but I could research more